ui/src/Groups/Counter/Counter.view.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Body, Subheading } from '../../Elements/Text/index.js';
+
+const Counter = (props) => {
+  const { title, subtitle, count, countLimit, className, ...others } = props;
+  const classesToApply = `Counter${className ? ` ${className}` : ''}`;
+
+  let countValue = count;
+  if (count < 0) {
+    countValue = '00';
+  } else if (count < 10) {
+    countValue = '0' + count;
+  } else if (count > countLimit) {
+    countValue = countLimit + '+';
+  }
+
+  return (
+    <div {...others} className={classesToApply}>
+      <div className={'Counter__title'}>
```

Comment:
Do we need the `<div>` wrapper? Could apply directly to the components (if needed at all)

Reply:
The Text components use `<span>`, so without the wrappers it ends up with the elements all sitting on the same row. I felt it was more correct to use `<div>` wrappers here rather than change the orientation with `flex-direction`. What do you think?

Reply:
Ah fair point, hadn't spotted that!